business_requirement_deliverable/security/ir.model.access.csv

```diff
@@ -1,6 +1,9 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_business_requirement_deliverable_user,business.requirement.deliverable.user,model_business_requirement_deliverable,business_requirement.group_business_requirement_user,1,1,1,1
 access_business_requirement_deliverable_manager,business.requirement.deliverable.manager,model_business_requirement_deliverable,business_requirement.group_business_requirement_manager,1,1,1,1
-
 access_business_requirement_resource_user,business.requirement.resource.user,model_business_requirement_resource,business_requirement.group_business_requirement_user,1,1,1,1
-access_business_requirement_resource_manager,business.requirement.resource.manager,model_business_requirement_resource,business_requirement.group_business_requirement_manager,1,1,1,1
\ No newline at end of file
+access_business_requirement_resource_manager,business.requirement.resource.manager,model_business_requirement_resource,business_requirement.group_business_requirement_manager,1,1,1,1
+access_deliverable_sale_report_user,business.requirement.deliverable.user,model_deliverable_sale_report,business_requirement.group_business_requirement_user,1,0,0,0
```

Comment:
about the view access, I think it should be available for both with same rights. It does not make sense to have 1,1,1,1 right?